### ISSUE TITLE
1.2.4 breaks webpack compilation

### ISSUE THREAD

**Issue by neonerd** [simov/slugify#16]
```
Module '"D:/xxx/node_modules/slugify/index"' resolves to a non-module entity and cannot be imported using this construct.
```

**Comment by neonerd** [comment]
Can be fixed by turning

```
import * as slugify from "slugify"
```

to

```
import  slugify from "slugify"
```

But this is still a breaking change for people using ES6 import notation and should've therefore been projected in the semver as 2.0.0.

**Comment by simov** [comment]
How can I reproduce this issue? The only thing added in 1.2.4 is the `index.d.ts` file.

**Comment by neonerd** [comment]
I'm not sure whether it was 1.2.4 or something before, the last working version was 1.2.1 for me.

I'll try to attach a reproducable setup tomorrow. :)

**Comment by zucher** [comment]
The Project https://github.com/Rebilly/ReDoc get the same error during "npm run build-dist"

**Comment by simov** [comment]
Like I said earlier the only thing changed was the TypeScript definition: https://github.com/simov/slugify/blob/master/index.d.ts 
Can you spot any issues with that file?

**Comment by jestho** [comment]
My guess would be that `export = slugify` should probably be `export default slugify`. That or `export declare function slugify(…)`

**Comment by simov** [comment]
Thanks @jestho! However I'm not that familiar with TypeScript. If anyone can test and see which one of the suggested syntax changes fixes the issue I'll add/merge it.

**Comment by neonerd** [comment]
You have a pull request fixing this, tested with newest Webpack / TS-Loader example.

Works both with

```
import slugify from "slugify"
```

and

```
import * as slugify from "slugify"
```

now.

**Comment by simov** [comment]
Everyone, the TypeScript definition was fixed in v1.2.6 ! :tada: :sparkles: 

Thank you! :+1:
